circus-train-housekeeping/src/test/java/com/hotels/bdp/circustrain/housekeeping/LegacyReplicaPathRepositoryTest.java

```diff
@@ -0,0 +1,118 @@
+package com.hotels.bdp.circustrain.housekeeping; /**
```

Comment:
Since we have a default "entity" implementation in the housekeeping project I think we should move all the DB CRUD type tests back there.